### PR TITLE
[C-1823] Fix twitter auth on android

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Add Blob support to AndroidManifest, twitter auth was breaking when trying to create a url from blob profile picture
* Adds error toast when any auth fails

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/screens/signon/ProfileAuto.tsx [hunk 1]
```diff
@@ -345,11 +348,11 @@ const ProfileAuto = ({ navigation }: ProfileAutoProps) => {
   ])
 
   useEffect(() => {
-    if (twitterError | instagramError | tikTokError) {
+    if (twitterError || instagramError || tikTokError) {
```
Comment (sliptype) [inline packages/mobile/src/screens/signon/ProfileAuto.tsx:351]: 🤦

File: packages/mobile/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -81,6 +81,11 @@
                 android:resource="@xml/file_paths"
             />
         </provider>
+        <provider
```
Comment (dylanjeffers) [inline packages/mobile/android/app/src/main/AndroidManifest.xml:84]: awesome find! how did you figure this out?

**Review by amendelsohn — APPROVED** [review]

File: packages/mobile/android/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Audius Music</string>
+    <string name="blob_provider_authority">co.audius.app.blob</string>
```
Comment (dylanjeffers) [inline packages/mobile/android/app/src/main/res/values/strings.xml:3]: blob_authority lmao

**Review by dylanjeffers — APPROVED** [review]
amazing finds and updates

File: packages/mobile/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (sliptype) [reply packages/mobile/android/app/src/main/AndroidManifest.xml:84]: Was getting `Cannot create URL for blob` error and found this issue https://github.com/charpeni/react-native-url-polyfill/issues/284

**Pull request merged**